content/react/concepts/routing/routing.md

```diff
@@ -34,48 +34,34 @@ Below, a `BrowserRouter` is used to map the navigation `<Link/>` components to a
 ```jsx
```

Comment:
I think that everything from the beginning of this file up to this line needs a second look since the text above this code snippet. It should be edited to include only language having to do with the latest version of React Router, v6. (for example, remove any mention of '<Switch/>`)